frontend-react/src/components/USLink.tsx

```diff
@@ -41,11 +106,11 @@ export const USNavLink = ({
 }: USNavLinkProps) => {
     return (
         <NavLink
-            to={href}
+            to={href || ""}
             className={({ isActive }) =>
                 isActive
-                    ? `usa-nav__link usa-current ${activeClassName || ""}`
-                    : `usa-nav__link ${className || ""}`
+                    ? classnames("usa-nav__link usa-current", activeClassName)
```

Comment:
Another benefit for `classnames` is that you can pass an object to toggle class names:

```jsx
className={ classnames("usa-nav__link", { "usa-current": isActive, [activeClassName]: isActive }) }
```